CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ The changes not yet present in any release are listed in this section.
 
 * Support for ASUS VivoBook 15 X510UA ([Alexander Agura](https://github.com/agura-lex)).
 
+### Changed
+
+* Replaced /etc/rc.local logic with a simple systemd service ([Alexander Agura](https://github.com/agura-lex)).
```

Comment:
Only new model items have the author name and profile link in brackets. Other changelog items are without any connection to the original author/s.

Reply:
Also, please, keep it consistent and use \*\<file-name\>\* for */etc/rc.local* file.